src/lib/account-profile/_account-profile-theme.scss

```diff
@@ -0,0 +1,11 @@
+@mixin vantage-account-profile-theme($theme) {
+  $foreground: map-get($theme, foreground);
+  $secondary-text: map-get($foreground, secondary-text);
+
+  vui-account-profile-menu {
+    .mat-list-base .mat-list-item .mat-line:nth-child(n + 2) {
+      color: $secondary-text;
+      @include mat-typography-level-to-styles($fontConfig, caption);
```

Comment:
Hmm I can see how this might conflict with passed in markup, but would work for most cases...